packages/newspringchurchapp/src/ui/WebBrowser/Browser.js

```diff
@@ -30,6 +30,8 @@ const Browser = {
     auth = { useRockCookie: false, useRockToken: false }
   ) => {
     const url = new URL(baseURL);
+    url._url = url.toString().slice(0, -1);
```

Comment:
@redreceipt The `new URL()` function adds a trailing slash to the end of the URL you pass to it. In the case of the Bug Report, it was putting a trailing slash after the first part of the querystring (that is part of the URL we pass it) and when it went to Safari, the link was escaping the "/" as "%2F" and causing issues. So this removes the trailing slash before anything else happens to the link. I tested all the links and didn't see any adverse effects. 

There might be a better way to remove the trailing slash (or to fix this issue entirely), but it's late and there's a good chance I'm just not thinking of it. Let me know if you think of a better way to handle this.